### PR TITLE
Changes to Step 2. Create or Obtain a Video Transcript (fixes DOC-1143)

### PR DESCRIPTION
**Pull request by lamagnifica**
@mslotkeEDX this changes "strongly recommended" to "required". I also did some reorganization and tweaking to this Step 2 section.
@srpearce , let me know if I introduced any errors!
@mhoeber , @catong , FYI
Nice to have this recommended --> required change in ASAP to support the PMs.

### CODE REVIEW

**Comment by lamagnifica** [comment]
@WatsonEmily too!

**Comment by mhoeber** [comment]
:+1:

File: en_us/course_authors/source/creating_content/create_video.rst [hunk 1]
```diff
@@ -22,10 +22,10 @@ Also see:
...
+To help course teams protect video assets, the edX video player hides video URLs from students by default. This functionality is in effect for all video files, including YouTube videos and videos that you have posted in other locations. Students can still download videos if you select the **Video Download Allowed** option for your Video components. For more information about posting non-YouTube videos online, see :ref:`Post the Video Online`. For more information about options that you can set when you create a video component, see :ref:`Video Advanced Options` under :ref:`Create a Video Component`.
```
Comment (srpearce) [inline en_us/course_authors/source/creating_content/create_video.rst]: Do you want to wrap this line?

Also, this paragraph (originally mine, so this is my fault) has both "Video component" and "video component." I get the sense we're moving away from capitalizing component types (which is more than fine with me!), but we should make sure this is consistent within topics.

File: en_us/course_authors/source/creating_content/create_video.rst [hunk 2]
```diff
@@ -101,33 +101,42 @@ The edX video player supports videos in .mp4, .webm, .mpeg, and .ogg format. **H
...
+of the video. You can allow students to download transcripts so that they
+can read them offline. You associate a transcript with a video when you
+create the Video component.
+
+A timed transcript automatically scrolls next to a video while it plays. When a
+student clicks a word in the transcript, the video begins to play where that
+word is spoken.
+
+**For a transcript to scroll next to a video automatically, it must be in .srt
+(SubRip Text) format.**
```
Comment (srpearce) [inline en_us/course_authors/source/creating_content/create_video.rst]: I think we should switch the order of this information - we tell them transcripts scroll automatically, but then we say "but only if  it's an .srt transcript." How about this?

We strongly recommend _(require?)_ that you use the SubRip Text (.srt) format for your transcripts. A transcript in the .srt format automatically scrolls next to its associated video as the video plays. A student can click a word in the transcript to jump to the point in the video where that word is spoken.

**Comment by srpearce** [comment]
A couple of comments; otherwise :+1:

File: en_us/course_authors/source/creating_content/create_video.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/course_authors/source/creating_content/create_video.rst]: - changed Video to lowercase in text throughout
- wrapped all unwrapped lines throughout the file
  They'll be in the next commit, after @mslotkeEDX's review

**Comment by WatsonEmily** [comment]
@mslotkeEDX - can you confirm this reflects the language you were requesting? 
Otherwise, :+1:

**Comment by lamagnifica** [comment]
(got thumbs from @mslotkeEDX in HipChat)

**Pull request merged**